tests/Writing/CopyMethodsTest.php

```diff
@@ -79,24 +85,6 @@ public function testWorkspaceCopy()
         $this->assertNotEquals($sfile->getPropertyValue('jcr:data'), $dfile->getPropertyValue('jcr:data'));
     }
 
-    public function testCopyPreserveChildOrder()
```

Comment:
Ah, I don't understand how I dit it :o I had to be tired :( 

I will revert it. I noticed that this test use php 5.4 short array syntax but was declared 5.3 support...